ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -34,7 +34,7 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
   def viewRecipe(id: String) = AuthAction { implicit request =>
     val recipe = db.getOriginalRecipe(id)
     db.insertUserEvent(UserEvent(request.user.email, request.user.firstName, request.user.lastName, id, UserEventAccessRecipeReadOnlyPage.name))
-    curatedRecipedEditor(recipe, editable = false)
+    curatedRecipedEditor(recipe, editable = false, userEmail = request.user.email, userFirstName = request.user.firstName, userLastName = request.user.lastName)
```

Comment:
Given all the properties are from the user object, would it be easier to only pass through the user object?